tests/integration/test_cluster_failover.py

```diff
@@ -35,7 +36,9 @@
 class TestClusterFailover(TestCase):
 
     def setUp(self):
-        with TypeDB.cluster_client(["localhost:11729", "localhost:21729", "localhost:31729"]) as client:
+        root_ca_path = os.environ["ROOT_CA"]
+        credential = TypeDBCredential("admin", "password", root_ca_path)
+        with TypeDB.cluster_client(["127.0.0.1:11729", "127.0.0.1:21729", "127.0.0.1:31729"], credential) as client:
```

Comment:
note to future self: must use IP to match built-in server credential, not `localhost`